coreos-devel/sdk-extras/sdk-extras-0.0.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 The CoreOS Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# This isn't meant to be installed directly. It mostly exists so we can better keep track of
+# dependencies within the SDK.
+
+EAPI=5
+
+DESCRIPTION="Meta ebuild for everything that isn't needed in the SDK, but might be useful"
+HOMEPAGE="http://coreos.com/docs/sdk/"
+SRC_URI=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE=""
+
+DEPEND="
```

Comment:
Is there a reason to make any of these build-time dependencies?  It seems like they could all be brought in at install time.

Reply:
Also, lists like this should be alphabetical.

Reply:
I'm not sure it really, matters, since this is mostly just to keep track of packages. I'll change it to RDEPEND though, no reason not to.